src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/DeviceTimeCheck.swift

```diff
@@ -15,20 +15,12 @@ final class DeviceTimeCheck: DeviceTimeCheckProtocol {
 
 	// MARK: - Init
 
-	init(store: AppConfigCaching & DeviceTimeChecking) {
+	init(store: AppConfigCaching & DeviceTimeChecking & AppFeaturesStoring) {
 		self.store = store
 	}
 
 	// MARK: - Protocol DeviceTimeCheckProtocol
```

Comment:
```suggestion
	// MARK: - Protocol DeviceTimeChecking
```

Seems to have been renamed later on, let's just clean it up while we're here 🧹

Reply:
done